website/versioned_docs/version-0.70/components-and-apis.md

```diff
@@ -67,6 +67,12 @@ These common user interface controls will render on any platform.
       <h3>Button</h3>
       <p>A basic button component for handling touches that should render nicely on any platform.</p>
     </a>
+  </div>
+   <div className="component">
+    <a href="./pressable">
+      <h3>Pressable</h3>
+      <p>A Core Component wrapper that can detect various stages of press interactions on any of its defined children</p>
```

Comment:
I love the idea of highlighting this component, but perhaps the verbiage could lead someone astray (what is a Core Component, is that another component we should be aware of?) Instead, I would slightly adjust the first half of the sentence to read:
```suggestion
      <p>A wrapper component that can detect various stages of press interactions on any of its children</p>
```